src/main/java/uk/gov/hmcts/dts/fact/services/admin/list/AdminAreasOfLawService.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.dts.fact.services.admin.list;
+
+import java.util.List;
+import static java.util.stream.Collectors.toList;
```

Comment:
Will this not introduce a checkstyle issue? The static import is amongst the non-static imports here. It looks like someone manually add the import here rather than using intellij to auto-import